pennylane/wires.py

```diff
@@ -123,6 +123,23 @@ def tolist(self):
         """
         return list(self.wire_tuple)
 
+    def at_index(self, idx):
+        """Returns the wire at index ``idx``.
```

Comment:
Tried to make it sound like one of the Python [list](https://docs.python.org/3/tutorial/datastructures.html) method docstrings 🤔 
```suggestion
        """Returns the wire representation at the given position in the wires object.
```